scanner.go

```diff
@@ -241,6 +261,9 @@ func (s *Scanner) parseEscapeSequence() (rune, error) {
 		}
 		return r, nil
 	default:
+		if '0' <= c && c <= '7' {
```

Comment:
Could you maybe instead write this as
```go
case '0', '1', '2', '3', '4', '5', '6', '7':
```
It's a bit silly but I think makes it a bit clearer what's going on.

Reply:
Done.